aws-iam-role-ci-user-manager/main.tf

```diff
@@ -7,6 +7,10 @@ module assume_role_policy {
   owner              = var.owner
   service            = var.service
   project            = var.project
+  ci_manager = {
+    sts_external_id   = var.sts_external_id
+    caller_account_id = var.caller_account_id
+  }
 }
 
 resource "aws_iam_role" "ci-manager" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "aws" in "required_providers"